src/data/resolvers/coinMachine.ts

```diff
@@ -198,13 +198,16 @@ export const coinMachineResolvers = ({
         // eslint-disable-next-line max-len
         const targetPerPeriodTokens = await coinMachineClient.getTargetPerPeriod();
 
+        const tokenPeriodBalance = await coinMachineClient.getTokenBalance();
```

Comment:
Can you please confirm what this method returns - `getTokenBalance`: is it the balance of tokens available for sale for the period? How can we then use it to decide if the sale has started? What if it's zero and we are just sold out for the period?

The logic (it's more for the `CoinMachine` component later as well) doesn't seem to be correct. I also remember Raul saying it was a bit more tricky than just that....

Reply:
`getTokenBalance` gets you the balance of the coin machine, so if the coin machine is empty then a sale hasn't been "started". So, yeah I should actually change the name and perhaps move it to a separate resolver as this is not the balance of a period but of the coin machine in general